app.js

```diff
@@ -224,6 +224,7 @@ if (app.get('env') === 'development') {
 
 // App configuration for all environments
 app.set('port', process.env.PORT || 3000);
+app.set('host', process.env.HOST || '0.0.0.0');
```

Comment:
how about something like this, and remove the line setting this on the app?
```javascript
  var server_params = {
            port :  process.env.PORT || 3000
        }
        
        if(process.env.HOST){
            server_params.host = process.env.HOST;
        }
```

Reply:
I do not see any pros and cons to this change unless the configuration is needed elsewhere in the application. In addition, the possibility of setting a default value is lost.

Reply:
In addition, you can not view the configuration in the log